File: Vibe.Domain/Employees/EmployeeBlank.cs
namespace Vibe.Domain.Employees
{
    public class EmployeeBlank
    {
        public Guid Id { get; set; }
        public String Name { get; set; }
        public String Phone { get; set; }
        public String Login { get; set; }
        public String Password { get; set; }
        public String Email { get; set; }

        public EmployeeBlank(Guid id, String name, String phone, String login, String password, String email)
        {
            Id = id;
            Name = name;
            Phone = phone;
            Login = login;
            Password = password;
            Email = email;
        }
    }
}

File: Vibe.BackOffice/vibe.backoffice.client/src/domain/employees/employeeBlank.ts
export class EmployeeBlank{
    constructor(
        public name: string | null, 
        public phone: string | null,
        public login: string | null,
        public password: string | null,
        public email: string | null, 
    ) {}

    public static getDefault(): EmployeeBlank {
        return new EmployeeBlank(null, null, null, null, null)
    }
}